src/Deployer.php

```diff
@@ -134,14 +140,14 @@ public function getOutput()
     /**
      * @param string $name
      * @return mixed
-     * @throws \InvalidArgumentException
+     * @throws InvalidArgumentException
```

Comment:
@GrahamCampbell i'm advocating the use of class names as you'd do if you were writing code. If you `use` a class or a namespace, then you should use that in the doc-block as well. 
Regarding your comment, i'm making it the same to read, you are making it hard to write.